airbyte-integrations/connectors/source-google-sheets/google_sheets_source/google_sheets_source.py

```diff
@@ -132,6 +130,7 @@ def read(
         sheet_to_column_name = Helpers.parse_sheet_and_column_names_from_catalog(catalog)
         spreadsheet_id = Helpers.get_spreadsheet_id(config["spreadsheet_id"])
 
+        row_batch_size = config['row_batch_size']
```

Comment:
```suggestion
        row_batch_size = config.get('row_batch_size', ROW_BATCH_SIZE)
```
Let the default batch in the code to not create backward compability and also let the default in the spec.